src/NumericalAlgorithms/LinearOperators/PowerMonitors.hpp

```diff
@@ -45,4 +45,64 @@ std::array<DataVector, Dim> power_monitors(const DataVector& input_data_vector,
                                            const Mesh<Dim>& mesh);
 /// @}
 
+/// @{
+/*!
+ * \ingroup SpectralGroup
+ * \brief Compute the negative log10 of the relative truncation error.
+ *
+ * Truncation error according to Eqs. (57) and (58) of Ref.
+ * \cite Szilagyi2014fna, i.e.,
+ *
+ * \f{align*}{
+ *  \mathcal{T}\left[P_k\right] = \log_{10} \max \left(P_0, P_1\right)
+ *   - \dfrac{\sum_{j} \log_{10} \left(P_j\right) w_j }{\sum_{j} w_j} ,
+ * \f}
+ *
+ * with weights
+ *
+ * \f{align*}{
+ *  w_j = \exp\left[ - (j - N_k + \dfrac{1}{2})^2 \right] .
+ * \f}
+ *
+ * where \f$ N_k \f$ is the number of modes or gridpoints in dimension k. Here
+ * the second term is a weighted average with larger weights toward the highest
+ * modes. The negative of this number should correspond to the number of digits
+ * resolved by the spectral expansion.
+ *
+ */
+double relative_truncation_error(const DataVector& input_power_monitors,
```

Comment:
remove `input_` (all `const&` function arguments are by definition inputs) and remove `s` at end. This is for a single variable.

I'm wondering if the first argument should really be a `ModalVector` since I think it holds the model coefficients of the L2 norm of `u`. @nilsvu thoughts?

Reply:
Sure, issue+code comment is fine, though it's not clear to me what makes changing it to a `ModalVector` in this PR difficult. Delaying it too long will make it a lot more work to refactor all the code that calls it...